### DISCUSSION TITLE
Privacy Concern: hide stars

### DISCUSSION THREAD

**Discussion by AnthonyMichaelTDM** [community/community#9231] [Other Feature Feedback, Questions, & Ideas]
I think a setting to hide repo's we've starred would be helpful for people concerned over privacy.

**Comment by ghost** [comment]
Hi, @AnthonyMichaelTDM 

So I hope to help. I created a gui - graphical user interface that exemplifies this.  My idea would be a checkbox where you could check and uncheck it. If I check the box, I keep people from seeing what I like. If you uncheck the box, you allow people to publicly see what you like.  An interesting and optional option would be "keep the stars private for those who follow my user profile". So if someone follows you, they don't have access to that information either. I think we can have an option like this:

#### image 1 
![image](https://user-images.githubusercontent.com/95204402/147841871-3539226f-1bad-4f7a-b582-496bb0d57db0.png)

#### or image 2
![image](https://user-images.githubusercontent.com/95204402/147842009-4258e198-4b29-46e4-af85-86d3b28b18ad.png)

Note: One thing I think and leave here: - Just as we can keep our email information private, why can't we keep our likes or stars private too?
